### PR TITLE
Update android.yml

### PR DESCRIPTION
**Pull request by MansiBhayade**
I have tried scheduling nightly build of app . Please do review it and suggest further changes needed to be made.

### CODE REVIEW

File: .github/workflows/android.yml [hunk 1]
```diff
@@ -1,9 +1,12 @@
...
 
 on:
-  push:
-    branches: [ beta ]
-  pull_request:
+   workflow_dispatch :
+   schedule :
+    - cron: '0 0 * * *'
+   push:
+      branches: [ beta ]
```
Comment (ukanth) [inline .github/workflows/android.yml:8]: There is no beta branch. It should be main. (currently it's master, but I will rename it)

File: .github/workflows/android.yml [hunk 2]
```diff
@@ -17,8 +20,15 @@ jobs:
       uses: actions/setup-java@v1
       with:
         java-version: 11
-    - name: Copy CI gradle.properties
-      run: mkdir -p ~/.gradle ; cp .github/ci-gradle.properties ~/.gradle/gradle.properties
+  #  - name: Copy CI gradle.properties
```
Comment (ukanth) [inline .github/workflows/android.yml:23]: this may be required ?

**Review by ukanth — COMMENTED** [review]
Renamed master to main

File: .github/workflows/android.yml (hunk 1 above)
Comment (MansiBhayade) [reply .github/workflows/android.yml:8]: okay. I will change it to main.

File: .github/workflows/android.yml (hunk 2 above)
Comment (MansiBhayade) [reply .github/workflows/android.yml:23]: Right now there is no such file as ci-gradle.properties to copy the contents. The default gradle settings are sufficient. So, I commented out this part of the code.

**Review by ukanth — APPROVED** [review]

**Pull request merged**